### PR TITLE
500.67 support. https://github.com/Koenkk/zigbee2mqtt/issues/2038

### PR DESCRIPTION
**Pull request by Koenkk**

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -857,6 +887,50 @@ const converters = {
...
         },
     },
+    command_ehanced_move_to_hue_and_saturation: {
+        cluster: 'lightingColorCtrl',
+        type: 'commandEnhancedMoveToHueAndSaturation',
+        convert: (model, msg, publish, options, meta) => {
+            const payload = {
+                action: getProperty(`enhanced_move_to_hue_and_saturation`, msg, model),
+                action_enhanced_hue: msg.data.enhancehue,
+                action_saturation: msg.data.saturation,
```
Comment (probutus) [inline converters/fromZigbee.js:897]: We have enhanced hue with values from 0..65535 and standard hue from 0 to 359 which is reflected in the variable name. Saturation is a value here from 0 to 255 instead  a percentage value? Is this intended

File: devices.js [hunk 2]
```diff
@@ -5322,6 +5322,20 @@ const devices = [
...
+    {
+        zigbeeModel: ['RemoteControl '],
+        model: '500.67',
+        vendor: 'Paulmann',
+        description: 'RGB remote control',
+        supports: 'action',
+        fromZigbee: [
+            fz.command_on, fz.command_off, fz.command_toggle, fz.command_step,
+            fz.command_move_to_color_temp, fz.command_move_to_color, fz.command_stop,
+            fz.command_move, fz.command_color_loop_set, fz.command_ehanced_move_to_hue_and_saturation,
```
Comment (probutus) [inline devices.js:5334]: is there any way to deal with the genOta messages?

Comment (Koenkk) [reply devices.js:5334]: We don't have to deal with them as we don't have an image.

File: converters/fromZigbee.js (hunk 1 above)
Comment (Koenkk) [reply converters/fromZigbee.js:897]: Will add `hue` and `saturation`.

**Pull request merged**